backend/tests/integration/schema_lifecycle/test_attribute_parameters_update.py

```diff
@@ -0,0 +1,252 @@
+        schema_root = SchemaRoot(**schema_step_01)
+        await load_schema_root(db=db, branch_name=default_branch.name, schema=schema_root, update_db=True)
+
+        # delete parameters from the LegacyThing schema attribute
+        query = """
+        MATCH (legacy_schema:SchemaNode)-[:HAS_ATTRIBUTE]->(:Attribute {name: "name"})-[:HAS_VALUE]->(av {value: "ThingLegacy"})
+        MATCH (legacy_schema)-[:IS_RELATED]-(:Relationship {name: "schema__node__attributes"})-[:IS_RELATED]-(schema_attr:SchemaAttribute)
+        MATCH (schema_attr)-[:HAS_ATTRIBUTE]->(params_attr:Attribute {name: "parameters"})
+        DETACH DELETE params_attr
+        """
+        await db.execute_query(query=query)
+
+    def _validate_schema_value_parameters(
+        self, schema: NodeSchema, regex: str | None, min_length: int | None, max_length: int | None
+    ):
+        value_attr = schema.get_attribute("value")
+        assert value_attr.regex == regex
+        assert value_attr.min_length == min_length
+        assert value_attr.max_length == max_length
+        assert isinstance(value_attr.parameters, TextAttributeParameters)
+        assert value_attr.parameters.regex == regex
+        assert value_attr.parameters.min_length == min_length
+        assert value_attr.parameters.max_length == max_length
+
+    async def test_schema_01_is_correct(self, db: InfrahubDatabase, default_branch: Branch, load_schema_01) -> None:
+        schema_branch = await registry.schema.load_schema_from_db(db=db, branch=default_branch.name)
+
+        legacy_schema = schema_branch.get_node(name=LEGACY_KIND, duplicate=False)
+        self._validate_schema_value_parameters(schema=legacy_schema, regex="old", min_length=3, max_length=4)
+
+        new_schema = schema_branch.get_node(name=NEW_KIND, duplicate=False)
+        self._validate_schema_value_parameters(schema=new_schema, regex="newnew", min_length=5, max_length=6)
+
+    async def test_schema02_load_update(
+        self,
+        client: InfrahubClient,
+        default_branch: Branch,
+        schema_step_02: dict[str, Any],
+    ):
+        success, response = await client.schema.check(schemas=[schema_step_02], branch=default_branch.name)
+        assert success
+        assert response == {
+            "diff": {
+                "added": {},
+                "changed": {
+                    LEGACY_KIND: {
+                        "added": {},
+                        "changed": {
+                            "attributes": {
+                                "added": {},
+                                "changed": {
+                                    "value": {
+                                        "added": {},
+                                        "changed": {
+                                            "regex": None,
+                                            "min_length": None,
+                                            "max_length": None,
```

Comment:
If I understood correctly, `min_length` and `max_length` should have changed, does having None value here means they changed?